src/app/components/tabs.js

```diff
@@ -0,0 +1,38 @@
+export default function tabs(container) {
+    const active = 'active';
+    const tabs = container.querySelectorAll('.tabs__button');
+    let tabActive = container.querySelector('.js-tabs .active');
```

Comment:
you already have the NodeList includes active tab, why you ask DOM again for it?

Reply:
look at contentActive